util-cache/src/main/contraband/remotecache.contra

```diff
@@ -0,0 +1,16 @@
+package sbt.util
+@target(Scala)
+
+type UpdateActionResultRequest {
+  actionDigest: sbt.util.Digest!
+  outputFiles: [xsbti.VirtualFile] @since("0.1.0")
+  exitCode: Int @since("0.2.0")
+  isExecutable: [Boolean] @since("0.3.0")
```

Comment:
Do we really need those now? How hard would it be to add new fields later?

Reply:
Yea, this is actually important to add this. I haven't gotten around to fully support this yet, but we need to add it here or maybe `xsbti.VirtualFile`, and one way or the other capture the executable flag, and test that both sbt can send the information and thaw it back correctly.

> How hard would it be to add new fields later?

Adding the field here won't be hard, but I want to make sure the build user can set this flag in a task, which might require some surgery.